frontend/src/js/store/organizationSlice/thunks.ts

```diff
@@ -266,6 +279,11 @@ export const getUserOrganization = createAsyncThunk(`${sliceName}/getUserOrganiz
     return Promise.all(tasks);
   });
 });
+export const getUserBilling = createAsyncThunk(`${sliceName}/getUserBilling`, (_, { dispatch }) => {
+  return Api.get(`${tenantadmApiUrlv2}/billing/profile`).then(res => {
+    dispatch(actions.setBillingProfile(res.data));
+  });
```

Comment:
I prefer braces around function body, so that it's quite easy to debug / add more instructions to the body, but for the sake of consistency with the rest of the codebase, I think we should [enable eslint rule ](https://eslint.org/docs/latest/rules/arrow-body-style). I will create a follow up ticket